sycl/test-e2e/KernelCompiler/kernel_compiler_spirv.cpp

```diff
@@ -0,0 +1,158 @@
+//==- kernel_compiler_spirv.cpp --------------------------------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// REQUIRES: ocloc
```

Comment:
Just a quick note, I made a normal SYCL app with a simple kernel and used `-fsycl-dump-device-code=./` when compiling it, then I fed that same .spv file to your kernel_bundle, was able to call `has_kernel(name)` and `get_kernel(name)` and pass it successfully to `parallel_for` . So that could be another way of testing, and without the ocloc dependency.